src/utils/hact/helpers/priority-codes.js

```diff
@@ -1,27 +1,27 @@
-export const mapPriorityCodeToHact = {
+export const priorityCodeCompletionTimes = {
+  // immediate
   1: {
     numberOfHours: 2,
     numberOfDays: 0,
-    priorityCodeHact: 1,
   },
```

Comment:
Matches https://github.com/LBHackney-IT/repairs-api-dotnet/blob/develop/RepairsApi/V2/Infrastructure/Migrations/20210118152301_SorCodeSeeding.cs#L48